packages/core/src/payment-provider-customer/payment-provider-customer-state.ts

```diff
@@ -0,0 +1,7 @@
+export default interface PaymentProviderCustomerState<T = any> {
```

Comment:
Do we need to use `any` here?

Reply:
To avoid generic prop drilling through CheckoutStoreState